postinstall.sh

```diff
@@ -6,9 +6,12 @@ set -x
 # If there is a libffmpegsumo, copy it over
 if test -f /opt/google/chrome/libffmpegsumo.so; then
 	cp /opt/google/chrome/libffmpegsumo.so node_modules/nw/nwjs/libffmpegsumo.so
+elif test -f /usr/lib/chromium/libffmpegsumo.so; then
+	cp /usr/lib/chromium/libffmpegsumo.so node_modules/nw/nwjs/libffmpegsumo.so
 else
 	echo "Couldn't locate libffmpegsumo.so during installation. Audio notifications will be disabled" 1>&2
-	echo "To repair this, please install Google Chrome and reinstall \`plaidchat\`" 1>&2
+	echo "To repair this, please install Google Chrome/Chromium and reinstall \`plaidchat\`" 1>&2
+	echo "Else you can copy the libffmpegsumo.so file in node_modules/nw/nwjs/" 1>&2
```

Comment:
This line won't be very straightforward to anyone using a non-development version. I would prefer to avoid it and keep it simple with a reinstall.